### PR TITLE
test(storage): fill out invoke tests

### PR DESCRIPTION
**Pull request by BrennaEpp**

### CODE REVIEW

File: storage/invoke_test.go [hunk 1]
```diff
@@ -121,8 +169,94 @@ func TestInvoke(t *testing.T) {
...
+			if test.expectFinalErr && got != test.finalErr {
+				s.Errorf("got %v, want %v", got, test.finalErr)
+			} else if !test.expectFinalErr && got != test.initialErr {
+				s.Errorf("got %v, want %v", got, test.initialErr)
+			}
+		})
+	}
+}
+
+func TestShouldRetry(t *testing.T) {
```
Comment (tritone) [inline storage/invoke_test.go:181]: Separating this out is much nicer, thanks!

File: storage/invoke_test.go [hunk 2]
```diff
@@ -121,8 +169,94 @@ func TestInvoke(t *testing.T) {
...
+		{
+			desc:        "googleapi.Error{Code: 400}",
+			inputErr:    &googleapi.Error{Code: 400},
+			shouldRetry: false,
+		},
+		{
+			desc:        "googleapi.Error{Code: 408}",
+			inputErr:    &googleapi.Error{Code: 408},
+			shouldRetry: false, // to be changed
+		},
```
Comment (tritone) [inline storage/invoke_test.go:253]: Could you add cases for retryable/non-retryable gRPC statuses?

**Review by tritone — COMMENTED** [review]
Couple little things, thanks for adding this!

File: storage/invoke_test.go (hunk 2 above)
Comment (BrennaEpp) [reply storage/invoke_test.go:255]: done! let me know if you think we should add more/all possible gRPC statuses

Comment (tritone) [reply storage/invoke_test.go:255]: looks good!

**Review by tritone — APPROVED** [review]

**Pull request merged**